### PR TITLE
Update to Django 2.0

### PR DESCRIPTION
**Pull request by anapaulagomes**
This PR solves #296. To check which warnings should be fixed I ran `python -Wall manage.py test`. Since we have some dependencies (at this point, just `django-simple-history`), I updated the libraries locally to make sure the changes are working and the tests are passing.

Since it's my first time contributing to Jarbas, it was a good way to know more about the code. :) I wasn't expecting to touch so many parts.

**How to test if it really works?**
Making sure the tests are passing. Question: should I do any manual tests? I ran it on my machine but I just played around.

**TODO**
- [ ] Wait until the `django-simple-history` release a new version with full support to Django 2.0

### CODE REVIEW

**Comment by cuducos** [comment]
Hi @anapaulagomes – thanks for this awesome PR!

While `django-simple-history` has no update I wish to share a worry with you all: we're already using [a fork](https://github.com/datasciencebr/jarbas/blob/master/requirements.txt#L1) of this package because of the lack of updates from the maintainers. Last year [we tried to get more maintainers involved](https://twitter.com/cuducos/status/870320609917620224) but for whatever reason the package has no updates in the last 8 months.

That said, we might track for forks that might be already updated for Django 2.0 and merge them to the branch we're using (this branch has some new lines to enhance i18n). I'm not confident the official repo will be updated soon – and I might be terribly wrong.

**TLDR** A nice way to contribute with Jarbas is to fork and update `django-simple-history` hahaha…

**Comment by anapaulagomes** [comment]
I saw the last comments on https://github.com/treyhunner/django-simple-history/pull/320 and it seems people are working on it. The last update was in November. Does your fork have different features (just trying to understand better)? I can work on the Django 2.0 update for your fork, no problem (I guess :P ).

**Comment by cuducos** [comment]
>  it seems people are working on it 

Great news then 🎉 Sorry about my skepticism/pessimism about it.

> The last update was in November

The last update on the repo was in November. But that last relase was in June (source: [GitHub](https://github.com/treyhunner/django-simple-history/releases) and [PyPI](https://pypi.python.org/pypi/rca-django-simple-history/1.9.1)). However if we're installing from GitHub, I prefer to keep my barely untouched fork than risking installing a `master` from other repo that eventually might include some breaking changes.

> Does your fork have different features (just trying to understand better)?

Just a single, but relevant, commit improving translations (that we use in Jarbas UI). [This commit is already in their `master` branch](https://github.com/treyhunner/django-simple-history/pull/281), but was not included in the latest PyPI release.
